Launcher/SimpleJobLauncher.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Akeneo\Bundle\BatchBundle\Launcher;
+
+use Akeneo\Bundle\BatchBundle\Entity\JobExecution;
+use Akeneo\Bundle\BatchBundle\Entity\JobInstance;
+use Akeneo\Bundle\BatchBundle\Job\JobRepositoryInterface;
+use Symfony\Component\Process\PhpExecutableFinder;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @author    Marie Bochu <marie.bochu@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/MIT MIT
+ */
+class SimpleJobLauncher implements JobLauncherInterface
+{
+    /** @var JobRepositoryInterface */
+    protected $jobRepository;
+
+    /** @var string */
+    protected $rootDir;
+
+    /** @var string */
+    protected $environment;
+
+    /** @var array */
+    protected $config = [];
+
+    /**
+     * Constructor
+     *
+     * @param JobRepositoryInterface   $jobRepository
+     * @param string                   $rootDir
+     * @param string                   $environment
+     */
+    public function __construct(JobRepositoryInterface $jobRepository, $rootDir, $environment)
+    {
+        $this->jobRepository     = $jobRepository;
+        $this->rootDir           = $rootDir;
+        $this->environment       = $environment;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function launch(JobInstance $jobInstance, UserInterface $user, $rawConfiguration = null)
+    {
+        $jobExecution = $this->create($jobInstance, $user);
+        $executionId  = $jobExecution->getId();
+        $pathFinder   = new PhpExecutableFinder();
+
+        if ($this->isConfigTrue('upload')) {
+            $rawConfiguration.= ' ' . addslashes(json_encode($jobInstance->getJob()->getConfiguration()));
+        }
+
+        $cmd = sprintf(
+            '%s %s/console akeneo:batch:job --env=%s %s %s %s %s >> %s/logs/batch_execute.log 2>&1',
+            $pathFinder->find(),
+            $this->rootDir,
+            $this->environment,
+            $this->isConfigTrue('email') ? sprintf('--email="%s"', $user->getEmail()) : '',
+            $jobInstance->getCode(),
+            $executionId,
+            !empty($rawConfiguration) ? sprintf('--config="%s"', $rawConfiguration) : '',
+            $this->rootDir
+        );
+
+        $this->launchInBackground($cmd);
+
+        return $jobExecution;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function launchInBackground($cmd)
```

Comment:
And yes, no need to put that in the interface or make it public. It's all about specific implementation.